### PR TITLE
move image-server out of status-im namespace

### PR DESCRIPTION
**Pull request by siddarthkay**
This PR moves `image_server.cljs` file out of status-im namespace into utils namespace.
All the references have also been updated.

status: ready

### CODE REVIEW

File: src/utils/image_server.cljs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(ns status-im.utils.image-server
+(ns utils.image-server
   (:require [quo.design-system.colors :as colors]))
```
Comment (flexsurfer) [inline src/utils/image_server.cljs]: please remove current-theme and quo.design-system.colors

Comment (flexsurfer) [reply src/utils/image_server.cljs]: probably timestamp as well, it should be in utils.date

Comment (siddarthkay) [reply src/utils/image_server.cljs]: I'll first check status-go implementation of passing theme property to get identicon and account image uris from media server.

Comment (siddarthkay) [reply src/utils/image_server.cljs]: I checked status-go and seems like the `theme` query url is passed for a reason : https://github.com/status-im/status-go/blob/develop/server/handlers.go and can not be safely removed without checking its consequences.. will investigate further

Comment (siddarthkay) [reply src/utils/image_server.cljs]: removing `theme` parameter from query url can impact identicon ring colour : 
https://github.com/status-im/status-go/blob/develop/server/handlers.go#L173-L185

Comment (flexsurfer) [reply src/utils/image_server.cljs]: i didn't mean to remove it from server, i just mean from `utils.image-server` namespace, sorry for confusion

Comment (flexsurfer) [reply src/utils/image_server.cljs]: `utils.image-server` should know nothing about `quo.design-system.colors`

Comment (siddarthkay) [reply src/utils/image_server.cljs]: Oh i see what you mean, thanks for clarifying :)

Comment (siddarthkay) [reply src/utils/image_server.cljs]: addressed over here : https://github.com/status-im/status-mobile/pull/15712/commits/d1eea8c121334e36dafcfae72d7ddd8f17d8e521

**Review by J-Son89 — APPROVED** [review]
LGTM 👍

**Comment by siddarthkay** [comment]
@status-im/mobile-qa : this should not create any issues but do have a quick look once if you can :)

**Review by flexsurfer — APPROVED** [review]

**Comment by pavloburykh** [comment]
> @status-im/mobile-qa : this should not create any issues but do have a quick look once if you can :)

hi @siddarthkay ! Could you please specify what exactly should be checked in scope of this PR? Upload of profile images/sending images or something else? Thanx!

**Comment by siddarthkay** [comment]
Hi @pavloburykh : This PR moved code from old namespace into new namespace. 
If there is an impact it should be to any place that uses image media server i.e Profile Images, Identicons and contact images.

**Comment by pavloburykh** [comment]
> Hi @pavloburykh : This PR moved code from old namespace into new namespace. If there is an impact it should be to any place that uses image media server i.e Profile Images, Identicons and contact images.

Thanx @siddarthkay ! Tested and ready to be merged.

**Pull request merged**